internal/services/fetchapi/fetchapi.go

```diff
@@ -0,0 +1,71 @@
+package fetchapi
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"sync"
+
+	pb "github.com/DIMO-Network/fetch-api/pkg/grpc"
+	"github.com/DIMO-Network/model-garage/pkg/cloudevent"
+	"github.com/DIMO-Network/rewards-api/internal/config"
+	"github.com/rs/zerolog"
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/credentials/insecure"
+)
+
+// FetchAPIService is a service for interacting with the Fetch API.
```

Comment:
nit: fetch isn't a very descriptive name; i know its already deployed but I dont love it

Reply:
Yeah there was long debate about this name. TLDR; I didn't come up with it but the idea is fetch API is a generic API for fetching any DIMO Data.